pkg/sku/azure_sku_handler.go

```diff
@@ -0,0 +1,58 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+package sku
+
+type AzureSKUHandler struct {
+	supportedSKUs map[string]GPUConfig
+}
+
+func NewAzureSKUHandler() *AzureSKUHandler {
+	return &AzureSKUHandler{
+		supportedSKUs: azureSupportedGPUConfigs,
+	}
+}
+
+func (a *AzureSKUHandler) GetSupportedSKUs() []string {
+	skus := make([]string, 0, len(azureSupportedGPUConfigs))
+	for sku := range azureSupportedGPUConfigs {
+		skus = append(skus, sku)
+	}
+	return skus
+}
```

Comment:
recreates a slice from a map, could be used elsewhere as well. move to utills.

Reply:
nvm this is ok

Reply:
This logic would be reused by other cloud provider implementations as well so I moved it to a helper file sku_utils